src/functionalTest/resources/scenarios/RIA-3278-appeal-fee-pay-later-EA-change-state-to-payment-pending-with-hearing.json

```diff
@@ -0,0 +1,45 @@
+{
+  "description": "RIA-3278 Change state to paymentPending for appeal type EA for pay later (with a hearing)",
+  "enabled": "{$featureFlag.isfeePaymentEnabled}",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "LegalRepresentative",
+    "input": {
+      "eventId": "submitAppeal",
+      "state": "appealStarted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "payForTheAppeal": "payLater",
+          "appealType": "refusalOfEu",
+          "decisionHearingFeeOption": "decisionWithHearing",
+          "homeOfficeDecisionDate": "{$TODAY-14}",
+          "feeCode": "FEE0238",
+          "feeDescription": "Appeal determined with a hearing",
+          "feeVersion": 2,
+          "feeAmount": 140.0,
+          "paymentStatus": "Payment due",
```

Comment:
I think these fields need to be removed from the functional test in order to see if you're getting back the correct response.

Reply:
Yes - the fee details are not required. However, to change the appeal state to Appeal_Submitted, it requires the payment status of "Paid"